tests/integration/test_k8s_with_cluster.py

```diff
@@ -150,8 +147,67 @@ def test_workflow_and_service_management_end_to_end_from_cli(
             delete_cluster_role_binding(workflow_sa_crb)
 
 
+def test_services_from_previous_deployments_are_deleted():
```

Comment:
I generally avoid trying to add anymore to this test because it does too much already (although it's got much simpler the last few iterations). It uses the five stage project so unless we were to change it to use the simpler one I think it would be best to keep it as is. 

Or create a branch with just the one service (no batch stages) for the second deploy? If we go for this i'll make the change as part of the remove workflow command ticket.

Reply:
Yeah - the second branch could be a trivial one.

I don't think this is essential. I agree with you when you say that we shouldn't stuff too much in each test. I'm focusing purely on test time, which may not be the right thing to do. In an idea world your current approach is the best one.

Reply:
I wonder whether to make the change in the other ticket and then leave the test commented out so we can reinstate it once we've got a better spec'd cluster